app/schemas/schemas.coffee

```diff
@@ -65,6 +65,15 @@ me.extendBasicProperties = (schema, linkFragment) ->
   schema.properties = {} unless schema.properties?
   _.extend(schema.properties, basicProps(linkFragment))
 
+# Basic props for new schemas using json draft 07 
+basicPropsNew = (linkFragment) ->
+  _id: me.objectId(links: [{rel: 'self', href: "/db/#{linkFragment}/{($)}"}], hidden: true)
```

Comment:
For my own understanding could you explain the link property here? Thank you!

Reply:
I am not really sure, but used the same thing as in existing `basicProps`.

Reply:
Reading about this, the `link` property describe the actions available to the client for a schema. In this case I think it means that this db entry can be accessed by `db/<linkFragment>/_id`.
Reference: Link Description Object section here: https://blog.apisyouwonthate.com/getting-started-with-json-hyper-schema-184775b91f

Reply:
Thanks! So it's a contract we have to ensure the server adheres to. That is very useful information! Thanks for sharing the link.